app/monitoring/CloudWatchMetrics.scala

```diff
@@ -0,0 +1,125 @@
+package monitoring
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchAsync
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest}
+import configuration.Config
+import play.api.libs.json.Json
+
+
+/**
+  * Created by mmcnamara on 12/07/2017.
+  */
+class CloudWatchMetrics(cloudWatchClient: AmazonCloudWatchAsync) extends TagAwareLogger {
+
+  val stage: String = Config.stage
```

Comment:
Minor point, but since we are listing the dependency on the cloud watch client explicitly, we might as well as well do the same for `stage` and provide it as a class parameter.